packages/labs/ssr/src/lib/module-loader.ts

```diff
@@ -253,8 +253,7 @@ export const resolveSpecifier = async (
     if (
       specifierMatches(specifier, 'lit') ||
       specifierMatches(specifier, 'lit-html') ||
-      specifierMatches(specifier, 'lit-element') ||
-      specifierMatches(specifier, '@lit/reactive-element')
+      specifierMatches(specifier, 'lit-element')
```

Comment:
Talked on chat: I think you should leave this and add `@lit/reactive-element` to ssr's dependencies

Reply:
Perfect suggestion. Worked! Thank you!